lib/src/internal_ble_manager.dart

```diff
@@ -185,20 +184,20 @@ class InternalBleManager
   }
 
   @override
-  Future<void> requestMtu(
+  Future<int> requestMtu(
       Peripheral peripheral, int mtu, String transactionId) {
     return _bleLib.requestMtu(peripheral, mtu, transactionId);
   }
 
   @override
   Future<List<Peripheral>> knownPeripherals(
       List<String> peripheralIdentifiers) {
-    return _bleLib.knownDevices(peripheralIdentifiers ?? []);
+    return _bleLib.knownDevices(peripheralIdentifiers);
```

Comment:
Might need confirmation

Reply:
the confirmation lays underneath ofc not here, but if one can pass in a null then they might as well just pass in an empty list. 
Null on input is not something desirable unless it actually carries a different semantic meaning.
so basically it's just guaranteed by the type system and not this extra safety code.